options/ansi/generic/string-stubs.cpp

```diff
@@ -474,6 +487,130 @@ void *mempcpy(void *dest, const void *src, size_t len) {
 }
 
 // GNU extensions.
+const char *strerrorname_np(int e) {
+	const char *s;
+	switch(e) {
+	case EAGAIN: s = "EAGAIN"; break;
```

Comment:
I'd prefer this to be done with a macro to avoid typos (there are several in the strings in this function), like so:
```cpp
	switch(e){
#define X(x) case x: s = #x; break;
	X(EAGAIN);
	X(EACCES);
	// ...
#undef X
	default:
	// ...
	}
```

Reply:
Fixed.